eng/Subsets.props

```diff
@@ -96,6 +96,18 @@
     <ProjectToBuild Include="$(RepoToolsLocalDir)regenerate-readme-table.proj" />
   </ItemGroup>
 
+  <ItemDefinitionGroup Condition="'$(CoreCLRConfiguration)' != ''">
+    <CoreClrProjectToBuild>
+      <Properties>Configuration=$(CoreCLRConfiguration)</Properties>
+    </CoreClrProjectToBuild>
+  </ItemDefinitionGroup>
+
+  <ItemDefinitionGroup Condition="'$(LibrariesConfiguration)' != ''">
+    <LibrariesProjectToBuild>
+      <Properties>Configuration=$(LibrariesConfiguration)</Properties>
```

Comment:
I didn't know you can define item properties without populating the item group first. But I'm definitely no msbuild expert so that if you know that this works, I'm fine with that.

Reply:
Yep! `ItemDefinitionGroup` allows you to specify default item metadata for a given item group.